src/scenes/Activities/scenes/EventHistory/index.js

```diff
@@ -292,6 +292,7 @@ export default class EventHistory extends Component {
         ],
         { field: 'endTime', direction: SortBy.Descending },
       );
+      this.navigating = true;
```

Comment:
prob not a big deal, but shouldn't we reset it back to false after navigating?

Reply:
I'm gonna test this, but I'm pretty sure when the component is unmounted, everything gets reset